src/carousel/carousel.scss

```diff
@@ -0,0 +1,193 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../text/index';
+
+@mixin iui-carousel {
+  @include iui-reset;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+@mixin iui-carousel-slider {
+  position: relative;
+  display: flex;
+  width: 100%;
+  @supports (aspect-ratio: 16 / 9) {
+    aspect-ratio: 16 / 9;
+  }
+  @include themed {
+    border-bottom: 2px solid t(iui-color-background-4);
+  }
+  overflow-x: scroll;
+  overflow-x: overlay;
+  counter-reset: item;
+  scroll-behavior: smooth;
+  scroll-snap-type: x mandatory;
+  -webkit-overflow-scrolling: touch;
+  scrollbar-color: transparent transparent;
+  scrollbar-width: none;
+  -ms-overflow-style: none;
+
+  &::-webkit-scrollbar {
+    width: 0;
+    height: 0;
+  }
+
+  &::-webkit-scrollbar-thumb {
+    background-color: transparent;
+    border: none;
+
+    &:hover {
+      background-color: transparent;
+    }
+  }
+
+  &-item {
+    position: relative;
+    flex-shrink: 0;
+    background-position: center;
+    background-size: cover;
+    counter-increment: item;
+    scroll-snap-align: center;
+    width: 100%;
+    box-sizing: border-box;
+    @supports (aspect-ratio: 16 / 9) {
+      aspect-ratio: 16 / 9;
+    }
+
+    &-title {
+      position: absolute;
+      left: 0;
+      bottom: 0;
+      padding: $iui-baseline $iui-sm;
+      box-sizing: border-box;
+      width: 100%;
+      @include themed {
+        background-color: rgba(0, 0, 0, t(iui-opacity-4));
+        color: t(iui-color-foreground-accessory);
+      }
+      @media (prefers-reduced-motion: no-preference) {
+        transition: background-color $iui-speed-fast ease;
+      }
+
+      &:hover {
+        @include themed {
+          background-color: rgba(0, 0, 0, t(iui-opacity-3));
+        }
+      }
+    }
+  }
+}
+
+@mixin iui-carousel-navigation {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: $iui-baseline * 3;
+
+  &-container {
+    display: flex;
+    flex-grow: 1;
+    white-space: nowrap;
+    overflow: hidden;
+    justify-content: center;
+  }
+
+  &-dots {
+    @include iui-reset;
+    display: flex;
+    align-items: center;
+    list-style: none;
+  }
+
+  &-dot {
+    @include iui-reset;
```

Comment:
does nothing on span and anchor elements

Reply:
Removed.